Packs/Citrix/Integrations/CitrixCloudEventCollector/CitrixCloudEventCollector.py

```diff
@@ -0,0 +1,205 @@
+        )
+
+        access_token = token_res.get("access_token")
+        if not access_token:
+            raise DemistoException("Failed to obtain access token from Citrix Cloud response.")
+
+        demisto.setIntegrationContext({ACCESS_TOKEN_CONST: access_token})
+        demisto.debug("access token created")
+        return access_token
+
+    def get_records(
+        self, start_date_time: str | None, end_date_time: str | None, continuation_token: str = None, limit: int = None
+    ):
+        # get access token value
+        integration_context = demisto.getIntegrationContext()
+        access_token = integration_context.get(ACCESS_TOKEN_CONST)
+
+        if not access_token:
+            access_token = self.request_access_token()
+
+        params = assign_params(
+            Limit=RECORDS_REQUEST_LIMIT,
+            continuationToken=continuation_token,
+            startDateTime=start_date_time,
+            endDateTime=end_date_time,
+        )
+
+        if limit and limit <= 200:
+            params["Limit"] = limit
+
+        headers = {
+            "Authorization": f"CwsAuth Bearer={access_token}",
+            "accept": "application/json",
+            "Citrix-CustomerId": self.customer_id,
+        }
+
+        demisto.info(f"Sending http request to get records with {params=}")
+        response = self._http_request(
+            "get", url_suffix="systemlog/records", headers=headers, params=params, ok_codes=[200, 401], resp_type="response"
+        )
+
+        if response.status_code == 401:
+            demisto.info("Access token expired; refreshing...")
+            access_token = self.request_access_token()
+            headers["Authorization"] = f"CwsAuth Bearer={access_token}"
+            demisto.info(f"Sending http request to get records with {params=}")
+
+            return self._http_request("get", url_suffix="systemlog/records", headers=headers, params=params)
+        else:
+            return response.json()
+
+    def get_records_with_pagination(self, limit: int, start_date_time: str | None, end_date_time: str | None = None):
+        records: list[dict] = []
+        continuation_token = None
+        raw_res = None
+
+        while len(records) < int(limit):
+            raw_res = self.get_records(
+                start_date_time=start_date_time, end_date_time=end_date_time, continuation_token=continuation_token, limit=limit
+            )
+            records.extend(raw_res.get("items", []))
+            continuation_token = raw_res.get("continuationToken")
+
+            if not continuation_token:
+                break
+
+        for record in records:
+            record["_time"] = record.get("utcTimestamp")
+
+        return records[:limit], raw_res
+
+
+""" HELPER FUNCTIONS """
+
+
+def get_events_command(client: Client, args: dict):  # type: ignore
+    limit = int(args.get("limit", "100"))
```

Comment:
```suggestion
    limit = int(args.get("limit", "10"))
```

Since its the get-command no reason to have so many events.